lightning-kmp-test-fixtures/src/commonMain/kotlin/fr/acinq/lightning/tests/TestConstants.kt

```diff
@@ -99,7 +104,9 @@ object TestConstants {
     }
 
     object Bob {
-        private val seed = ByteVector32("0202020202020202020202020202020202020202020202020202020202020202")
+        private val entropy = Hex.decode("0202020202020202020202020202020202020202020202020202020202020202")
+        val mnemmonics = MnemonicCode.toMnemonics(entropy)
+        val seed = MnemonicCode.toSeed(mnemmonics, "").toByteVector32()
```

Comment:
It's still not corrected for Bob.
You fixed the fact that in the previous commit, Alice incorrectly used Bob's mnemonic and ended up with the same seed as Bob, which our tests didn't catch (we probably should test somewhere that Alice and Bob have different seed and nodeId).

```suggestion
        val mnemonics = MnemonicCode.toMnemonics(entropy)
        val seed = MnemonicCode.toSeed(mnemonics, "").toByteVector32()
```